aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -191,7 +207,7 @@ The app currently displays a simple form to send messages. Nothing happens when
 1. Execute the following command at the project root:
 
     ```console
-    npm install @microsoft/signalr
+    npm i @microsoft/signalr @types/node
     ```
 
     The preceding command installs the [SignalR TypeScript client](https://www.npmjs.com/package/@microsoft/signalr), which allows the client to send messages to the server.
```

Comment:
```suggestion
    The preceding command installs:
    
     * The [SignalR TypeScript client](https://www.npmjs.com/package/@microsoft/signalr), which allows the client to send messages to the server.
     * The TypeScript type definitions for Node.js, which enables compile-time checking of Node.js types.
```

The type definitions package is installed here too. I recommend adding an explanation similar to what I've provided above.